packages/botonic-plugin-contentful/tests/contentful/contents/handoff.test.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  ContentCallback,
+  ContentType,
+  HandoffAgentId,
+  HandoffQueue,
+  SPANISH,
+} from '../../../src'
+import { testContentful, testContext } from '../contentful.helper'
+
+const TEST_HANDOFF_AGENT_ID_MAIN_ID = '6I5SudoItjGmxrVZPr0qeG'
+const TEST_HANDOFF_QUEUE_MAIN_ID = '6A7D4ssRYuLufjvl1pnOzV'
+const TEST_QUEUE_ID = '62ILnVxLHOEp7aVvPMpCO8'
+
+describe('Contentful Handoff', () => {
+  test('TEST: contentful handoff with queue as destination', async () => {
```

Comment:
When I link the handoff with a Queue that contains an Scheduler it fails: 
```
Error calling CMS.handoff with locale 'es' on 'handoff' with id '6A7D4ssRYuLufjvl1pnOzV'. 
Error loading Scheduler '71twiV4wcaFwhK6tSYuIKy'. Broken reference? Not published?
```
I've checked and the reference is okey, @dpinol any idea on why isn't it woking?